src/styles/imports/Colors.less

```diff
@@ -5,8 +5,9 @@
 @purple-semi: rgba(135, 54, 224, 0.08);
 @red: #f85640;
 @red-semi: rgba(248, 86, 64, 0.08);
+@light-blue: #CEFFED;
 // @yellow: #ffc803;
-@yellow: #e3973e;
```

Comment:
this is our correct yellow color - see beaker figma
feel free to add a new variable for the more orange all-stars color

Reply:
was mistake sorry, Fixed it.